.github/workflows/release.yml

```
@@ -17,24 +17,38 @@ jobs:
           # Requires "Read and Write access to code" permission
           token: ${{ secrets.RELEASE_ACTION_ACCESS_TOKEN }}
 
-      - name: Create Tag
-        id: tag_version
+      - name: Fetch latest release version tag
+        id: fetch_tag
         run: |
-          # Extracts the latest version tag, bumps the minor version
-          LATEST_VERSION=$(git tag --sort=-authordate --merged=main | head --lines 1)
+          LATEST_VERSION=$(curl -s https://api.github.com/repos/cowprotocol/services/releases/latest | jq -r '.tag_name')
           if ! [[ "$LATEST_VERSION" =~ ^v[0-9]+\.[0-9]+\..* ]]; then
             echo "Invalid tag format, cannot bump version of: $LATEST_VERSION"
             exit 1
           fi
-          MAJOR=$(echo $LATEST_VERSION | cut -d. -f1)
-          MINOR=$(echo $LATEST_VERSION | cut -d. -f2)
+          echo "latest=$LATEST_VERSION" >> $GITHUB_OUTPUT
+
+      - name: Check for changes
+        run: |
+          git diff --exit-code || echo "CHANGES_DETECTED=true" >> $GITHUB_ENV
+
+      - name: No changes detected
+        if: env.CHANGES_DETECTED != 'true'
+        run: echo "No updates necessary. Dependencies are already up-to-date."
```

Our dependencies are most certainly not up-to-date in this repo. 😅 
```suggestion
        run: echo "No commits since previous release. Skipping release."
```

---

Thanks, it was a wrong copy-paste 😄